MeshLib/MeshGenerators/LayeredVolume.cpp

```diff
@@ -58,6 +59,7 @@ bool LayeredVolume::createRasterLayers(const MeshLib::Mesh &mesh,
 	this->_minimum_thickness = minimum_thickness;
 	_nodes = MeshLib::copyNodeVector(bottom->getNodes());
 	_elements = MeshLib::copyElementVector(bottom->getElements(), _nodes);
+	_materials.insert(_materials.begin(), _elements.size(), 0); 
```

Comment:
don't understand what you mean. the existing elements of the 2d mesh are carried over as "0th" layer of the 3d mesh. material properties reflect this by assigned "0" as material id for said elements.